Assets/MixedRealityToolkit.Examples/Demos/Input/Scenes/DisablePointers/DisablePointersExample.cs

```diff
@@ -130,6 +130,11 @@ void Update()
             SetToggleHelper<SpherePointer>(GrabToggle, "GrabToggle", InputSourceType.Hand);
             SetToggleHelper<PokePointer>(PokeToggle, "PokeToggle", InputSourceType.Hand);
             SetToggleHelper<GGVPointer>(GazeToggle, "GazeToggle", InputSourceType.Hand);
+
+            DrawDebugRay(InputSourceType.Hand, Handedness.Left);
```

Comment:
I think it's great to have a test that shows getting the data working even without pointers, however it would be nice to actually pull it out into a separate demo scene. As it stands, it's a bit hidden / not discoverable.